extensions/validation/src/test/java/org/mule/extension/validation/BasicValidationTestCase.java

```diff
@@ -193,8 +191,6 @@ public void successfulAll() throws Exception
     {
         MuleEvent event = getAllEvent(VALID_EMAIL, VALID_URL);
         MuleEvent responseEvent = runFlow("all", event);
-
-        // validation was successful and payload was untouched
```

Comment:
We shouldn't add comments to the test, perhaps a better solution would be to change the method name to reflect what are we trying to prove.